app/code/Magento/InventoryCatalog/Test/Integration/InventoryCatalog/Model/Source/Validator/PreventDisableValidatorTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Test\Integration\InventoryCatalog\Model\Source\Validator;
```

Comment:
Move to
https://github.com/magento-engcom/msi/tree/2.3-develop/app/code/Magento/InventoryCatalogApi/Api

Reply:
Thanks, I'll move it to `\Magento\InventoryCatalogApi\Test\Api\DisableDefaultSourceTest` this namespace. It is ok now?